base/js/admin.js

```diff
@@ -542,7 +586,7 @@ var sowbForms = window.sowbForms || {};
 			});
 			var repeaterHtml = repeaterObject.html().replace(/_id_/g, $nextIndex);
 
-			var readonly = typeof $el.attr('readonly') != 'undefined';
+			var readonly = typeof $el.attr('readonly') !== 'undefined';
```

Comment:
I wonder if we still need to use `typeof` anymore. Maybe `_.isEmpty` or just `!== undefined`. I think `typeof` was a hack for older browsers.